### DISCUSSION TITLE
Mod Load order

### DISCUSSION THREAD

**Discussion by Jibby2121** [xivdev/Penumbra#190] [Q&A]
I started using this over tex tools and am enjoying figuring it out however I was wondering about the following.

Mod priority. 0 is highest then 1 , 2 etc? Correct?

If mods like for example Owls eyes of Eorzea. It has 2 installs. The main one and the Endwalker update. I'm assuming I can leave both at load order 0 or do I need to make the endwalker update one the lower in the priority since the base one is the main one? Or the other way around since the addon modifies the base install?

Its the same with other mods that I have where I have the base then an addition to modify the base files.

I have a lot of red conflicts but they all seem to be working fine? So Am I worrying to much?

Or is it just a matter of trial and error and I should be going for all whit no conflicts instead of red and yellow (not suyre what yellow means yet haha)

**Comment by reiichi001** [comment]
Outside of conflicts, all mods should be priority 0 (lowest). Big number = bigger priority and should only be used if there are conflicts. You'll see penumbra change the color when the conflict is marked as resolved. 

Unlike textools, you don't really have to worry about installing mods in any order at all. Ordering in your mod list is purely for organizational purposes. 

Penumbra should have some built in tool tips to explain a number of functions, colors included!
